index.bs

```diff
@@ -806,7 +802,7 @@ To <dfn>handle an incoming message</dfn> given a [=WebSocket connection=]
        [=WebSocket connection=] be |connection|, and remove |connection| from
        the set of [=WebSocket connections not associated with a session=].
 
-    1. Let |response| be a new map matching the <code>CommandResponse</code>
+    1. Let |response| be a new [=/map=] matching the <code>CommandResponse</code>
```

Comment:
Some line below there is a `map` entry like `If parsed is a map ...`.  I assume this should also be linked? If yes, maybe there are other instances as well.

Reply:
I think that there is still one instance left which should be correctly linked.

`The return value of this algorithm is a map between event names and
contexts.`

Reply:
Similar for `set`, mostly when I search for ` set.`

`|browsing contexts| is a set because a [=shared worker=] can be associated with multiple contexts.`

`contains |connection|, remove |connection| from that set.`

`Let |event names| be an empty set.`

And others.

Reply:
FWIW I don't think we need to link text that's just informative e.g. `the return value of this alogrithm is a map`. But I'll fix the rest.

Reply:
Sounds fair. Thanks!